ios/PolkadotVault/Screens/Scan/DynamicDerivations/AddDerivedKeysView.swift

```diff
@@ -265,22 +242,6 @@ extension AddDerivedKeysView {
             isPresented = false
             onCompletion(.onCancel)
         }
-
-        private func continueCreation() {
-            derivedKeysService.createDerivedKeys(
-                dynamicDerivationsPreview.keySet.seedName,
-                seedsMediator.getSeed(seedName: dynamicDerivationsPreview.keySet.seedName),
-                keysToImport: dynamicDerivationsPreview.keySet.derivations
-            ) { [weak self] result in
-                switch result {
-                case .success:
-                    self?.onSuccess()
-                case let .failure(error):
-                    self?.presentableError = .importDynamicDerivedKeys(content: error.localizedDescription)
```

Comment:
You didn't remove this function importDynamicDerivedKeys() yet? Is that intentional?

Reply:
Yeah, should remove that as well, thanks